<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewColDlg</class>
 <widget class="QDialog" name="NewColDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="dataTypeLabel">
       <property name="text">
        <string>Data type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboDType">
       <item>
        <property name="text">
         <string>CHAR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DECIMAL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DOUBLE PRECISION</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FLOAT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BIGINT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NUMERIC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SMALLINT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TIMESTAMP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VARCHAR</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="columnNameLabel">
       <property name="text">
        <string>Column name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edtColumnName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="fillLabel">
       <property name="text">
        <string>Fill</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="edtFill"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewColDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewColDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
